RELEASING.md

```diff
@@ -33,32 +33,53 @@ release is announced.
+      documents how to do this with the `release-note` tag.
+   1. Run [generate-release-notes.py](https://github.com/tigera/process/blob/master/releases/generate-release-notes.py)
+      with appropriate settings for MILESTONE and GITHUB_TOKEN environment variables (e.g. `export MILESTONE='Calico v2.5.0'`):
+   1. Copy/paste the contents of `generated-release-notes.md` into the GitHub Release Notes.
 1. Edit the [Calico Docs Custom Search Engine](https://cse.google.com/).
    1. Navigate to: search engine -> Search Features -> Refinements -> Add
-   2. Add a new refinement name: vX.Y
-   3. Navigate to: Setup -> Basics
-   4. Under "Sites to search", select "Add", for the url use `docs.projectcalico.org/vX.Y`
-   5. Choose vX.Y from the "Label" dropdown.
+   1. Add a new refinement name: vX.Y
+   1. Navigate to: Setup -> Basics
+   1. Under "Sites to search", select "Add", for the url use `docs.projectcalico.org/vX.Y`
+   1. Choose vX.Y from the "Label" dropdown.
 1. Edit `_config_dev.yml` to exclude the previous release.
 
 ### Promoting a release candidate to a final release
 1. Add a new `<option>` entry to the `<span class="dropdown">` in `_layouts/docwithnav.html` file. This step should NOT be performed until testing of the release is complete.
-1. Modify the redirect in `/index.html` to point to your new release.. 
+1. Modify the redirect in `/index.html` to point to your new release.
 1. Move the section for the release in `_data/versions.yaml` to the top of the file so that it will be the 'Latest Release'.
 1. Run `make add_redirects_for_latest VERSION=vX.Y` to update the redirects.
 1. Test the changes locally then open a pull request, make sure it passes CI and get it reviewed.
-1. Create a git tag, docker images, and release file by running `make -C calico_node release`. Follow the instructions to push the images but DO NOT PUSH THE TAG YET.
+1. Create a git tag, docker images, and release file by running `make -C calico_node release`.
+   Follow the instructions to push the images but DO NOT PUSH THE TAG YET.
 1. Now merge the PR - this will cause the live docs site to be updated (after a few minutes).
 1. Push the tag.
-1. Create a GitHub release on the Calico repo, and upload the release file (calico_node/release-<VERSION>.tgz)
+1. Create a GitHub release on the Calico repo, upload the release file (calico_node/release-<VERSION>.tgz), and post the updated
+   release notes (see above for details).
 
 ### Performing a "patch" release
 Patch releases shouldn't include any new functionality, just bug fixes (expect during pre-release testing).
-1. Add the new version to `_data/versions.yaml` (Remember to add the date in the description)
+1. Add the new version to `_data/versions.yaml`:
+   - Add the date in the description.
+   - Make sure new release candidate versions are added to the top of the new release section (otherwise
```

Comment:
release candidate? This section is about patch releases...

Reply:
Its the same process for patches and major/minor versions. But since I encountered this problem while creating the second release candidate, I'll move this to the major/minor section.

Reply:
okay, I just pushed a new version which relocates this admonition to the major/minor release section.